toolkit/Makefile

```
@@ -12,14 +12,17 @@ toolkit_root := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 SCRIPTS_DIR  ?= $(toolkit_root)/scripts
 
 # Build targets
+##help:var:CONFIG_FILE:<config_path>=Path to image configuration file to use.
 ifeq ($(origin CONFIG_FILE), undefined)
 CONFIG_FILE             = $(toolkit_root)/imageconfigs/core-efi.json
-$(warning CONFIG_FILE is undefined, defaulting to toolkit's core-efi.json.)
+#$(warning CONFIG_FILE is undefined, defaulting to toolkit's core-efi.json.)
 endif
 CONFIG_BASE_DIR         ?= $(dir $(CONFIG_FILE))
 PACKAGE_BUILD_LIST      ?=
+##help:var:PACKAGE_REBUILD_LIST:<spec_list>=List of spec basenames to force rebuild.
 PACKAGE_REBUILD_LIST    ?=
 PACKAGE_IGNORE_LIST     ?=
+##help:var:SRPM_PACK_LIST:<spec_list>=List of spec basenames to build into SRPMs; if empty, all found specs will be packed.
```

Similar comment as for `PACKAGE_REBUILD_LIST`:
```suggestion
##help:var:SRPM_PACK_LIST:<spec_list>=List of space-separated spec folders inside "SPECS_DIR" to analyze for the build. If empty, all specs inside the "SPECS_DIR" will be analyzed. Example: SRPM_PACK_LIST="kernel go which".
```

---

I'd also add something like "exclusively analyze" here to indicate we won't look at the other packages.